sdk/communication/azure-communication-chat/azure/communication/chat/_shared/utils.py

```diff
@@ -17,8 +17,8 @@
 
 
 def _convert_datetime_to_utc_int(expires_on):
-    epoch = time.mktime(datetime(1970, 1, 1).timetuple())
-    return epoch-time.mktime(expires_on.timetuple())
+    epoch = time.localtime().tm_gmtoff
```

Comment:
Since its not `epoch`, could we rename the variables?
```suggestion
    offset_from_utc = time.localtime().tm_gmtoff
    return time.mktime(input_datetime.timetuple()) + offset_from_utc
```